net/mac-telnet/patches/010-openwrt-name.patch

```diff
@@ -0,0 +1,11 @@
+--- a/config.h
++++ b/config.h
+@@ -42,7 +42,7 @@
+ #define PLATFORM_NAME "BSD/OS"
+
+ #elif defined(linux) || defined(__linux__)
+-#define PLATFORM_NAME "Linux"
++#define PLATFORM_NAME "OpenWRT"
```

Comment:
Correct capitalization should be `OpenWrt`

Reply:
Done!